web/themes/new_weather_theme/templates/partials/daily-forecast-list-item.html.twig

```diff
@@ -20,24 +20,27 @@ only be a nighttime period.
         <span class="display-block">{{ periods[0].monthAndDay }}</span>
       </time>
 
-      {% for period in periods %}
-        {% include '@new_weather_theme/partials/daily-high-low.html.twig' with { 'period': period } %}
-      {% endfor %}
+      {% include '@new_weather_theme/partials/daily-high-low.html.twig' with { 'periods': periods } %}
     </div>
     <div class="grid-row grid-gap">
-      {% for period in periods %}
-        {% if period.isDaytime %}
-          {% set label = "Day" | t %}
-          {% set lineColor = "yellow" %}
-          {% set marginUnit = 3 %}
-        {% else %}
-          {% set label = "Night" | t %}
-          {% set lineColor = "indigo" %}
-          {% set marginUnit = 1 %}
-        {% endif %}
-        <div class="daily-forecast-summary-area grid-col-12 tablet:grid-col-6 margin-bottom-{{ marginUnit }} tablet:margin-bottom-1">
+        {% for period in periods %}
+            {% set marginUnit = 3 %}
+            {% set tabletCol = 6 %}
+            {%  if period.isOvernight %}
+                {% set label = "Overnight" | t %}
+                {% set lineColor = "midnight-indigo" %}
+                {% set tabletCol = 12 %}
+            {% elseif period.isDaytime %}
+                {% set label = "Day" | t %}
+                {% set lineColor = "yellow" %}
+            {% else %}
+                {% set label = "Night" | t %}
+                {% set lineColor = "indigo" %}
+                {% set marginUnit = 1 %}
+            {% endif %}
+        <div class="daily-forecast-summary-area grid-col-12 tablet:grid-col-{{ tabletCol }} margin-bottom-{{ marginUnit }} tablet:margin-bottom-{{ marginUnit }}">
```

Comment:
@eric-gade it's very close but the tricky thing is that  it should be `tablet:margin-bottom-3` for overnight, but `tablet:margin-bottom-1` for day and night

Reply:
Hoping https://github.com/weather-gov/weather.gov/pull/1017/commits/0d9da6aad202956b2158588537da9965c9793d33 takes care of it

Reply:
👍 Looks good now!